docsrc/2-steps/your-first-fable-project.md

```diff
@@ -12,26 +12,29 @@ Now we're ready, let's start a new project using Fable!
 
 <li>
 
-### Use Fable template
+### Use a Fable template
 
-The easiest way to get starting with Fable is by using Fable official template.
+The easiest way to get started with Fable is to use one of the Fable official templates. First list them, then pick a template with a name that starts with "fable" to create a new project.
 
-1. `dotnet new -i "Fable.Template::*"`
-2. `dotnet new fable -n MyFirstFableProject`
-3. `cd MyFirstFableProject`
+1. `dotnet new -i 'Fable.Template::*'`
+2. `mkdir MyFirstFableProject ; cd MyFirstFableProject`
```

Comment:
In general, we prefer to show one command at a time.

Reply:
The existing version already has three numbered commands. How about accept this pull request and deal with that issue separately?